PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -170,6 +173,28 @@ extension HomeViewController {
             let cell: TopicChipCell = collectionView.dequeueCell(for: indexPath)
             cell.configure(model: TopicChipPresenter(title: slate.name, image: nil))
 
+            return cell
+        case .recentSaves(let objectID):
+            let cell: ItemsListItemCell = collectionView.dequeueCell(for: indexPath)
+            
+            guard let presenter = model.presenter(for: objectID) else {
+                return cell
+            }
+
+            cell.backgroundConfiguration = .listPlainCell()
+            cell.layer.borderWidth = 0.5
+            cell.layer.borderColor = UIColor(.ui.grey6).cgColor
+            cell.layer.cornerRadius = 16
```

Comment:
What do you think about putting this bit of styling code in the view itself? Then we could add a `style` property to the view model and set it to `.bordered` (or some such name) for this use case.

Something along the lines of 

```swift
extension ItemsListItemCell {
  enum CellStyle {
    case bordered
    case plain
  }
  struct Model {
    // ... omitting existing properties
    let style: CellStyle // default this to .plain
  }
}
```